pkgs/by-name/mo/modrinth-app-unwrapped/package.nix

```
@@ -163,7 +164,9 @@ rustPlatform.buildRustPackage {
     ];
     maintainers = with lib.maintainers; [ getchoo ];
     platforms = lib.platforms.linux ++ lib.platforms.darwin;
-    # this builds on architectures like aarch64, but the launcher itself does not support them yet
-    broken = !stdenv.hostPlatform.isx86_64;
+    # This builds on architectures like aarch64, but the launcher itself does not support them yet.
+    # Darwin does not experience this issue.
+    # See https://github.com/modrinth/code/issues/776#issuecomment-1742495678
+    broken = !stdenv.hostPlatform.isx86_64 && stdenv.isLinux;
```

It would be better to check if it's *not* Darwin, as it is the exception here and theoretically the launcher can build for other platforms

```suggestion
    broken = !stdenv.hostPlatform.isx86_64 && !stdenv.hostPlatform.isDarwin;
```